### PR TITLE
ROX-24161: do not use `roxctl --password` in scripts

### PR DESCRIPTION
**Pull request by parametalol**
### Description

Follow-up of #12381 (passing passwords to curl).

The goal is to not expose the password, passed via command line arguments, in the process list.

* Passed the password from `ROX_ADMIN_PASSWORD` variable via pipe to the pod context for the case when `roxctl` is executed in the central pod like in `kubectl exec deploy/central -- roxctl`.
* Renamed `ROX_PASSWORD` to `ROX_ADMIN_PASSWORD`: allows for removing the `--password` parameter from `roxctl` invocations.
* Renamed `STACKROX_PASSWORD` to `ROX_ADMIN_PASSWORD`: just for consistency.

:warning: This may require developers to change their environment from using `ROX_PASSWORD` to `ROX_ADMIN_PASSWORD`.

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [x] modified existing tests

#### How I validated my change

WIP

### CODE REVIEW

File: deploy/k8s/sensor.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ fi
 echo "CLUSTER set to $CLUSTER"
 
 if [[ -z "${ROX_ADMIN_PASSWORD}" ]]; then
-    export ROX_ADMIN_PASSWORD="${ROX_PASSWORD:-}"
+    export ROX_ADMIN_PASSWORD="${ROX_ADMIN_PASSWORD:-}"
```
Comment (porridge) [inline deploy/k8s/sensor.sh:23]: This block seems be a no-op, except in the case `ROX_ADMIN_PASSWORD` is either unset, or set to empty value but not exported. Either way, it does not seem to provide much value?

File: scale/profiler/launch-profiler.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
```
Comment (porridge) [inline scale/profiler/launch-profiler.sh:1]: Are we sure bash is available in the environment where the profiled is being launched?

**Review by porridge — APPROVED** [review]
LGTM overall, but please make sure the prow jobs run.

File: scale/profiler/launch-profiler.sh (hunk 2 above)
Comment (0x656b694d) [reply scale/profiler/launch-profiler.sh:1]: We've got `${BASH_SOURCE[0]}` in line 18. That made me think that bash was available.

**Review by dvail — APPROVED** [review]
LGTM from the UI side

**Pull request merged**